src/python/era_5g_object_detection_common/era_5g_object_detection_common/fps_test_detector.py

```diff
@@ -43,3 +46,24 @@ def process_image(self, frame):
             self.frames = 0
             self.start_time = time.time()
             return [([0, 0, 0, 0], fps, 0, "fps")]
+        
+    def process_images(self, frames: List[np.array]):
+        """
+        Wrapper around process_image to simulate batched processing.
+
+        Args:
+            frames (list(np.array)): Batch with frames
+
+        Returns:
+            list(list(tuple(bbox[], fps_value, class_id, class_name))):
+                List with values that would be returned by individual calls to
+                process_image.
+        """
+        
+        batched_results = []
+
+        for frame in frames:
```

Comment:
Just a tip, the code could be rewritten as `return [self.process_image(frame) for frame in frames]` ;)

Reply:
well, ok then, if you prefer it this way :)